src/client/eppo-client.ts

```diff
@@ -154,6 +163,28 @@ export default class EppoClient {
     this.isObfuscated = isObfuscated;
   }
 
+  async withOverrides(overridePayload: string | undefined): Promise<EppoClient> {
```

Comment:
```suggestion
  async withOverrides(overridePayload?: string): Promise<EppoClient> {
```

Reply:
We actually want to force the consumer to provide an overridePayload. It just allows undefined to make it more ergonomic to work with (see other comment)

Reply:
So the thinking is they just pass through whatever comes from our extension, and we handle the parsing?

Reply:
Yeah, that's what I was thinking. We could also provide middleware helpers for popular frameworks.